test/Templates.Test/Helpers/AddFirewallExclusion.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Diagnostics;
+using System.IO;
+
+namespace Templates.Test.Helpers
+{
+    internal class AddFirewallExclusion : IDisposable
+    {
+        private bool _disposedValue = false;
+        private readonly string _exclusionPath;
+
+        public static IDisposable Create(string exclusionPath)
+        {
+            return new AddFirewallExclusion(exclusionPath);
+        }
+
+        private AddFirewallExclusion(string exclusionPath)
+        {
+            if (!File.Exists(exclusionPath))
+            {
+                throw new FileNotFoundException();
```

Comment:
Very minor point, but we could pass `exclusionPath` to the `FileNotFoundException` constructor so it gives a clearer error message.